File: Kefche/Server/Web/Kefche/Hubs/Chess/ChessTurnInfo.cs
namespace Kefche.Hubs.Chess
{
    using System.Collections.Generic;

    public class ChessTurnInfo
    {
        public string SessionId { get; set; }

        public string ConnectionId { get; set; }
        public string Fen { get; set; }
        public List<object> MoveList { get; set; }
    }
}

File: Kefche/Client/ClientApp/src/app/models/chess-turn-info.ts
export class ChessTurnInfo{
    constructor(connectionId: string, sessionId: string, fen: string, moveList: Array<any>) {
        this.ConnectionId = connectionId;
        this.SessionId = sessionId;
        this.Fen = fen;
        this.MoveList = moveList;
    }

    ConnectionId: string;
    SessionId: string;
    Fen: string;
    MoveList: Array<any>;
}